lib/jwt_signed_request/verify.rb

```diff
@@ -50,8 +50,11 @@ def secret_key
       @secret_key ||= stored_key.fetch(:key) { raise MissingKeyIdError }
     end
 
+    JWT_BEARER_REGEX = /\A(Bearer\s)?([^*]+)\z/
```

Comment:
Does the trailing whitespace after `Bearer` need to be one character or can it be `\s+`?

Reply:
The Bearer authentication scheme is borrowed from the OAuth 2.0 authorization framework ([rfc6750](https://tools.ietf.org/html/rfc6750)) and denotes the format as follows:

```
b64token    = 1*( ALPHA / DIGIT / "-" / "." / "_" / "~" / "+" / "/" ) *"="
credentials = "Bearer" 1*SP b64token
```

In layman terms, the format is "Bearer" + a minimum of 1 space character with an infinite upper bound followed by the base64 encoded value (see [RFC2616, section 2](https://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html) for definitions). While most won't use multiple whitespace conditions, I think @zubin is correct here and we should account for it. Apologies on the oversight.

```suggestion
    JWT_BEARER_REGEX = /\A(Bearer\s+)?([^*]+)\z/
```